apps/hash-graph/lib/graph/src/shared/identifier/mod.rs

```diff
@@ -24,14 +24,17 @@ pub enum GraphElementId {
 // WARNING: This MUST be kept up to date with the enum variants.
 //   We have to do this because utoipa doesn't understand serde untagged:
 //   https://github.com/juhaku/utoipa/issues/320
```

Comment:
Does this need updating (and other similar ones)

Reply:
I was able to derive it by adding a schema for `BaseUri`: 1dfe10810e8ffa893be9fe0e2a997c744609d5c0